### ISSUE TITLE
Determine correct response to encryption questionnaire

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#75]
When the app is uploaded to Apple there is a questionnaire that needs to be completed. Attached are the screen shots of how it is currently being answered. We need to make sure the questions are being answered correctly as incorrect answers may be unnecessary.

**NOTE** Canada is except from US cryptography export restrictions. As such, if we make this application available in the Canada and US store only then we may not need to do any further reporting on this matter.

[Apple Docs](https://developer.apple.com/documentation/security/complying_with_encryption_export_regulations)
[US Gov Requirements](https://www.bis.doc.gov/index.php/policy-guidance/encryption)

These [Screen Shots](https://app.zenhub.com/files/429950091/a87520b7-8adf-40c1-97d2-df55575c3075/download) are now I think they should be answered. I'm currently answering **NO** to the first screen because we are not live in the iTunes Store and we're not technically exporting anything due to my **NOTE** above.

**Comment by jleach** [comment]
@CharlesMacpherson @swcurran Let's touch base closer to release and make sure we're answering the cryptography questions accurately. Added you to the assignees so we don't loose track.

**Comment by swcurran** [comment]
I think that question 2 ("qualify for exemption") may not be correct ("No" currently), but perhaps should be "Yes", as I think (c) is applicable.

(c) does not cover encryption, but that is covered in the next screen, and we are using "standard" encryption, so I think we are good there.  Likewise, the next screen is correct ("Yes"), as we do encryption beyond that of the Apple OS.

My recommendation is that for today, we leave it as Canada/US only, and that Charles/Oliver/John take the question to legal for at least some evaluation.  I think there must be a lot of information about this, particularly in the context of government apps vs. private industry apps.

Ultimately, our target audience is in BC, so it's not the end of the world either way. It would be nice from a marketing perspective to allow others to try our app.

**Comment by CharlesMacpherson** [comment]
I agree with Stephen. I thought we fit into (c). Our focus right now is BC, so I am fine with keeping it North America only for the time being. Let's do that. We can revisit with legal if access outside of North America becomes important for us.

**Comment by jleach** [comment]
James will put some docs together and we can add them to the Bifold repo. Waiting...

**Comment by cvarjao** [comment]
@jleach , did we ever get any further information on this subject?
